g3appdriver/src/main/java/org/rmj/g3appdriver/GRider/Database/Repositories/AppTokenManager.java

```diff
@@ -0,0 +1,32 @@
+package org.rmj.g3appdriver.GRider.Database.Repositories;
+
+import android.app.Application;
+import android.os.AsyncTask;
+
+import org.rmj.g3appdriver.GRider.Database.AppDatabase;
+import org.rmj.g3appdriver.GRider.Database.DataAccessObject.DRawDao;
+import org.rmj.g3appdriver.GRider.Database.Entities.ETokenInfo;
+
+
+public class AppTokenManager {
+
+    private final DRawDao dao;
+
+    public AppTokenManager(Application instance){
+        AppDatabase db = AppDatabase.getInstance(instance);
+        this.dao = db.RawDao();
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.